pkg/update.go

```diff
@@ -5,13 +5,16 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+type LabelSet map[string]struct{}
```

Comment:
Is it using a `struct{}` preferrable over using a `bool`?

Reply:
According to, https://yourbasic.org/golang/implement-set/, it's a minor performance optimization since an empty struct allocates no bytes.